yelp/data/fetchData.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+readonly DATA_URL='https://linqs-data.soe.ucsc.edu/public/psl-examples-data/yelp.zip'
```

Comment:
The permissions on these files is a bit off.
It should be 755 for directories (which is correct here) and 644 for files (they currently are 755).